### ISSUE TITLE
backstage version bump is not possible - createCardExtension should be imported from @backstage/plugin-home-react

### ISSUE THREAD

**Issue by jillesmc** [drodil/backstage-plugin-toolbox#55]
Hi @drodil.
It seems that now `createCardExtension` should be imported from `@backstage/plugin-home-react` instead of `@backstage/plugin-home`

This issue is this as it's impacting the ability to upgrade to the latest version of Backstage.

This line
https://github.com/drodil/backstage-plugin-toolbox/blob/57ff723f71b242403307451bb340756e3bda8c78/plugins/toolbox/src/plugin.ts#L7

Should be changed to
```typescript
 import { createCardExtension } from '@backstage/plugin-home-react'; 
```
Just like you did in the [qeta](https://github.com/drodil/backstage-plugin-qeta/blob/main/plugins/qeta/src/plugin.ts#L8) plugin

You can see similar and related issues across other repositories
- https://github.com/RoadieHQ/roadie-backstage-plugins/issues/1114
- https://github.com/backstage/backstage/issues/20196

**Comment by drodil** [comment]
True, have to get this fixed.

**Comment by drodil** [comment]
Fixed in 4d934a7739f4d4f9e28bdf9dd4e6ed2e4ad60765

**Comment by drodil** [comment]
Published version 1.11.5 with the fix. Thanks again for reporting @jillesmc ! Closing this issue now.
